frontend/dockerfile/instructions/parse.go

```diff
@@ -282,6 +282,7 @@ func parseAdd(req parseRequest) (*AddCommand, error) {
 	flChmod := req.flags.AddString("chmod", "")
 	flLink := req.flags.AddBool("link", false)
 	flKeepGitDir := req.flags.AddBool("keep-git-dir", false)
+	flParents := req.flags.AddBool("parents", false)
```

Comment:
I think we probably want to check `parentsEnabled` here instead of in `convert.go` - that means we can keep the convert logic the same, all that changes is we'd conditionally parse it depending on labs/stable channel, which is how we've done heredocs and other syntax changes in the past.

Reply:
Done